nemo_curator/synthetic/prompts.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Nit long term we should probably move to using `jinja` for prompt templates, this is fine for now.